packages/liveblocks-react-ui/src/components/AiTool.tsx

```diff
@@ -174,7 +175,11 @@ const noop = () => {};
 export const AiTool = Object.assign(
   forwardRef<HTMLDivElement, AiToolProps>(
     ({ children, title, icon, className, ...props }, forwardedRef) => {
-      const { status, toolName } = useAiToolInvocationContext();
+      const {
+        status,
+        toolName,
+        [kInternal]: { execute },
```

Comment:
I wouldn't mind exposing this on the context as an `isAutoExecuting: boolean` if you would prefer that.

Reply:
I think my preference would be to wait before someone asks for it, since in non-generic cases (unlike us with `AiTool`) you would know if a tool has an `execute` method or not, since you'd be the one defining it just above `render`.